cpp/include/cudf/io/parquet_metadata.hpp

```diff
@@ -37,23 +36,8 @@ namespace io {
  * @file
  */
 
-//! Parquet I/O interfaces
-namespace parquet {
-/**
- * @brief Basic data types in Parquet, determines how data is physically stored
- */
-enum class TypeKind : int8_t {
-  UNDEFINED_TYPE       = -1,  // Undefined for non-leaf nodes
-  BOOLEAN              = 0,
-  INT32                = 1,
-  INT64                = 2,
-  INT96                = 3,  // Deprecated
-  FLOAT                = 4,
-  DOUBLE               = 5,
-  BYTE_ARRAY           = 6,
-  FIXED_LEN_BYTE_ARRAY = 7,
-};
-}  // namespace parquet
+//! Parquet physical `Type`
+using cudf::io::parquet::Type;
```

Comment:
probably clearer not to use an alias

Reply:
Using `cudf::io::parquet::Type` directly in `parquet_column_schema` unfortunately fails docs build for some reason. Hence this alias.